js/README.md

```diff
@@ -17,50 +17,165 @@
   under the License.
 -->
 
-### Installation
+# [Apache Arrow](https://github.com/apache/arrow) in JS
 
-From this directory, run:
+[![Build Status](https://travis-ci.org/graphistry/arrow.svg?branch=master)](https://travis-ci.org/graphistry/arrow)
+[![Coverage Status](https://coveralls.io/repos/github/graphistry/arrow/badge.svg)](https://coveralls.io/github/graphistry/arrow)
 
-``` bash
-$ npm install   # pull dependencies
-$ npm run lint -- <filename>  # run tslint
-$ npm run build # build typescript (run tsc and webpack)
-$ npm run test  # run the unit tests (node.js only)
+Loading big native dataframes in JavaScript is finally awesome. `apache-arrow` provides an easy, modern, and efficient zero-copy JS interface to parse, iterate, and access [Apache Arrow](https://github.com/apache/arrow) columnar data on CPUs (GPU support via [GoAI](http://gpuopenanalytics.com/) is occurring in parallel).
+
+## install
+
+`npm install apache-arrow`
+
+# What's Arrow?
+
+Apache Arrow is a columnar memory layout specification for encoding vectors and table-like containers of flat and nested data. The Arrow spec aligns columnar data in memory to maximize caches and take advantage of the latest SIMD (Single input multiple data) and GPU operations on modern processors.
+
+Apache Arrow is the emerging standard for large in-memory columnar data ([Spark](https://spark.apache.org/), [Pandas](http://wesmckinney.com/blog/pandas-and-apache-arrow/), [Drill](https://drill.apache.org/), ...). By standardizing on a common interchange format, big data systems can reduce the costs and friction associated with cross-system communication.
+
+# Related Projects
+
+* [GoAI](http://gpuopenanalytics.com/) -- Arrow standard extensions for GPUs
```

Comment:
This reference is fine, but we should find a more accurate way to describe the GoAI project, since Arrow has extensions for GPUs, too.